### PR TITLE
`EVP_PKEY_{en,de}capsulate.pod`: fix glitches and add some detail and hints

### PR DESCRIPTION
**Pull request by DDvO**
SSIA

##### Checklist

- [x] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by DDvO** [comment]
Ouch, many (clearly unrelated) CI failures.
Hope this will get resolved QUIC-kly.

**Review by paulidale — APPROVED** [review]

**Review by t8m — CHANGES_REQUESTED** [review]

File: doc/man3/EVP_PKEY_decapsulate.pod [hunk 1]
```diff
@@ -13,14 +13,16 @@ EVP_PKEY_decapsulate_init, EVP_PKEY_auth_decapsulate_init, EVP_PKEY_decapsulate
...
-                          unsigned char *secret, size_t *secretlen,
+                          unsigned char *unwrapped, size_t *unwrappedlen,
                           const unsigned char *wrapped, size_t wrappedlen);
 
 =head1 DESCRIPTION
 
 The EVP_PKEY_decapsulate_init() function initializes a private key algorithm
 context I<ctx> for a decapsulation operation and then sets the I<params>
 on the context in the same way as calling L<EVP_PKEY_CTX_set_params(3)>.
+Note that I<ctx> is usually is produced using L<EVP_PKEY_CTX_new_from_pkey(3)>,
```
[the remark this answers is not in the document]
Comment (DDvO) [reply doc/man3/EVP_PKEY_decapsulate.pod]: fixed

**Review by t8m — APPROVED** [review]

**Comment by DDvO** [comment]
Thanks @paulidale and @t8m for your swift approvals.
Pauli, could you please re-confirm after the nit fix.

**Review by paulidale — APPROVED** [review]

**Comment by DDvO** [comment]
Merged to master, 3.0, and 3.1.
Thanks @t8m and @paulidale for your quick approvals.

**Pull request closed without merging**